src/models/split_transfers.py

```diff
@@ -84,6 +84,16 @@ def _process_native_transfers(
                     penalty_total -= transfer.amount_wei
                     continue
             self.eth_transfers.append(transfer)
+
+        if indexed_slippage is not None:
+            # This is the case when solver had Negative Payment
+            # and did not appear in the list of unprocessed_native transfers
+            # slippages were expected drained!
+            # TODO - Deal with this!
+            raise ValueError(
+                "This is a scenario where solver has no payment but also has negative slippage!"
+            )
+
```

Comment:
This is the place where preexisting overdrafts would cause a problem. If not addressed, we would have skipped slippage computation. Luckily we are now catching it with this Value Error.